src/Storage/custom/hybrid-2019-03-01/New-AzStorageAccount.ps1

```diff
@@ -0,0 +1,338 @@
+        ${AsJob},
+    
+        [Parameter(DontShow, HelpMessage='Wait for .NET debugger to attach')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Wait for .NET debugger to attach
+        ${Break},
+    
+        [Parameter(DontShow, HelpMessage='SendAsync Pipeline Steps to be appended to the front of the pipeline')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline
+        ${HttpPipelineAppend},
+    
+        [Parameter(DontShow, HelpMessage='SendAsync Pipeline Steps to be prepended to the front of the pipeline')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+        ${HttpPipelinePrepend},
+    
+        [Parameter(DontShow, HelpMessage='The URI for the proxy server to use')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use
+        ${Proxy},
+    
+        [Parameter(DontShow, HelpMessage='Credentials for a proxy server to use for the remote call')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call
+        ${ProxyCredential},
+    
+        [Parameter(DontShow, HelpMessage='Use the default credentials for the proxy')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy
+        ${ProxyUseDefaultCredentials}
+    )
+    
+    process {
+        if ($PSBoundParameters.ContainsKey("StorageEncryption")) {
+            $null = $PSBoundParameters.Add("EncryptionKeySource", "Microsoft.Storage")
+            $null = $PSBoundParameters.Remove("StorageEncryption")
+        }
+        if ($PSBoundParameters.ContainsKey("KeyVaultEncryption")) {
+            $null = $PSBoundParameters.Add("EncryptionKeySource", "Microsoft.Keyvault")
+            $null = $PSBoundParameters.Remove("KeyVaultEncryption")
+        }
+        $null = Az.Storage.internal\New-AzStorageAccount @PSBoundParameters
+        if ($PSBoundParameters.ContainsKey("InputObject")) {
+            $PSBoundParameters["InputObject"] | Az.Storage.internal\Get-AzStorageAccountProperty
+        } else {
+            Az.Storage.internal\Get-AzStorageAccountProperty -Name $PSBoundParameters["Name"] -ResourceGroupName $PSBoundParameters["ResourceGroupName"]
+        }
```

Comment:
Is this replacing the command, or are there additional aprameter sets?

Right now it looks like there should at least be an additional parameter set where neither Storage nor KeyVault encryption is used.

Reply:
If no switch is used (`-StorageEncryption` or `-KeyVaultEncryption`) it would have no encryption. I found another parameter set to be unnecessary